README.md

```diff
@@ -51,11 +51,18 @@ Each release of Dapr CLI includes various OSes and architectures. These binary v
    * For Linux/MacOS - `/usr/local/bin`
    * For Windows, create a directory and add this to your System PATH. For example create a directory called `c:\dapr` and add this directory to your path, by editing your system environment variable.
 
-### Install Dapr on your local machine (standalone)
+### Install Dapr on your local machine (self-hosted)
+
+In self-hosted mode, dapr can be installed with docker(recommended) or without a docker dependency.
```

Comment:
In essence, this is not about with docker or without docker, or with these containers or without. The init command provides a "developer environment" that allows a user to get started easily. It does so using Docker so there is a dependency in the default installation. If a user does not want to use docker or does not wish to have that dev env setup for them, Dapr offers a minimal init (or slim). I would add this kind of explanation here to better articulate it